src/retrieve-validation/node.ts

```diff
@@ -50,6 +50,8 @@ const retrieveInNode = async (
 			}
 		);
 
+		req.write(parameters);
```

Comment:
Shouldn't this be wrapped in a check for the `POST` method? E.g.,

```suggestion
		if (method === 'POST') {
			req.write(parameters);
		}
```

Reply:
Ah, yes, good catch.